mk/uvmt/uvmt.mk

```diff
@@ -63,6 +63,7 @@ MKDIR_P = mkdir -p
 # Compile compile flags for all simulators (careful!)
 WAVES        ?= 0
 SV_CMP_FLAGS ?= "+define+$(CV_CORE_UC)_ASSERT_ON"
+SV_CMP_FLAGS = ""
```

Comment:
Does this mean the assertions are not compiled or used in simulation?

Reply:
> What is the rule in core-v-verif about those asserts which are inside the some design RTL files?

There is no rule about this, but in general we try to use assertions in both formal and simulation.